hooks/pre-push

```diff
@@ -50,7 +50,7 @@ bash scripts/run_linter.sh
 check_test_status "Linter"
 
 echo "Running karma tests"
-bash scripts/run_karma_tests.sh --browsers PhantomJS
```

Comment:
Yep -- but it turns out that PhantomJS couldn't use es6, so it was giving errors for the backtick character in template files. So it seemed better to go back to Chrome, rather than have pre-push testing that didn't work at all...